config/routes.rb

```diff
@@ -0,0 +1,85 @@
+Rails.application.routes.draw do
+  get '/' => 'users#index'
+
+  resources :markets
```

Comment:
Great use of resources for markets here! You could also utilize resources for routes in products, sales, and vendors as well. Check out this for nesting resources: http://guides.rubyonrails.org/routing.html#nested-resources For example: ```resources :markets do
    resources :vendors
  end```